src/picasso-definition/interactions/tap-navigation-panel.js

```diff
@@ -0,0 +1,23 @@
+const tap = ({ eventName = 'tap-navigation-panel' }) => ({
```

Comment:
What is the purpose of this interaction?

Reply:
(discussed)